install_config/advanced_ldap_configuration/sssd_for_ldap_failover.adoc

```diff
@@ -0,0 +1,425 @@
+applicable to a setup in a container.
+
+[[sssd-prerequisites-for-authenticating-proxy-setup]]
+== Prerequisites for Authenticating Proxy Setup
+
+. Before starting setup, you need to know the following information about your
+LDAP server.
++
+* Whether the directory server is powered by
+http://www.freeipa.org/page/Main_Page[FreeIPA], Active Directory, or another
+LDAP solution.
++
+* The Uniform Resource Identifier (URI) for the LDAP server (for example,
+ldap.example.com).
++
+* The location of the CA certificate for the LDAP server.
++
+* Whether the LDAP server corresponds to RFC 2307 or RFC2307bis for user groups.
++
+. Prepare the VMs:
++
+* *_proxy.example.com_*: A VM to use as the authenticating proxy. This machine must
+have at least SSSD 1.12.0 available, which means a fairly recent operating
+system. This topic uses a Red Hat Enterprise Linux 7.2 server for its examples.
++
+* *_openshift.example.com_*: A VM to use to run {product-title}.amp
+
+[NOTE]
+====
+These VMs can be configured to run on the same system, but for the examples used
+in this topic they are kept separate.
+====
+
+[[sssd-phase-1-certificate-generation]]
+== Phase 1: Certificate Generation
+
+. To ensure that communication between the authenticating proxy and
+{product-title} is trustworthy, create a set of Transport Layer Security (TLS)
+certificates to use during the other phases of this setup. We will start by
+using the auto-generated certificates created as part of running:
++
+====
+----
+# openshift start \
+    --public-master=https://openshift.example.com:8443 \
+    --write-config=/etc/origin/
+----
+====
++
+Among other things, this generates a *_/etc/origin/master/ca.{cert|key}_*. Use
+this signing certificate to generate keys to use on the authenticating proxy.
++
+====
+----
+# mkdir -p /etc/origin/proxy/
+# oadm ca create-server-cert \
+    --cert='/etc/origin/proxy/proxy.example.com.crt' \
+    --key='/etc/origin/proxy/proxy.example.com.key' \
+    --hostnames=proxy.example.com \
+    --signer-cert=/etc/origin/master/ca.crt \
+    --signer-key='/etc/origin/master/ca.key' \
+    --signer-serial='/etc/origin/master/ca.serial.txt'
+----
+====
++
+This command automatically adds all valid IP addresses.
++
+[IMPORTANT]
+====
+Ensure that any host names that might need to access the proxy are listed, or
+the HTTPS connection will fail.
+====
+
+. Generate the API client certificate that the authenticating proxy will use
+to prove its identity to {product-title}. This is necessary and prevents
+malicious users from impersonating the proxy and sending fake identities.
+
+. Create a new CA to sign this client certificate:
++
+====
+----
+# oadm ca create-signer-cert \
+  --cert='/etc/origin/proxy/proxyca.crt' \
+  --key='/etc/origin/proxy/proxyca.key' \
+  --name='openshift-proxy-signer@`date +%s`' \
+  --serial='/etc/origin/proxy/proxyca.serial.txt'
+----
+====
++
+`date +%s` makes the signer unique. You can use any name you prefer, however.
```

Comment:
For the formal documentation, let's take`` `date +%s` ``out. It's kind of confusing. Maybe just have:

```
# oadm ca create-signer-cert \
  --cert='/etc/origin/proxy/proxyca.crt' \
  --key='/etc/origin/proxy/proxyca.key' \
  --name='openshift-proxy-signer@UNIQUESTRING' \
  --serial='/etc/origin/proxy/proxyca.serial.txt'
```

and remind them to make UNIQUESTRING something actually unique.